Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Themes/InteractableThemeBase.cs

```diff
@@ -11,89 +11,108 @@ namespace Microsoft.MixedReality.Toolkit.UI
     /// <summary>
     /// Base class for themes
     /// </summary>
-
     public abstract class InteractableThemeBase
     {
         public Type[] Types;
         public string Name = "Base Theme";
-        public List<InteractableThemeProperty> ThemeProperties = new List<InteractableThemeProperty>();
-        public List<InteractableCustomSetting> CustomSettings = new List<InteractableCustomSetting>();
+        public List<ThemeStateProperty> StateProperties = new List<ThemeStateProperty>();
+        public List<ThemeProperty> Properties = new List<ThemeProperty>();
         public GameObject Host;
         public Easing Ease;
-        public bool NoEasing;
         public bool Loaded;
-        public string AssemblyQualifiedName;
 
-        private bool hasFirstState = false;
+        /// <summary>
+        /// TODO: Troy
```

Comment:
These todos don't have any description, can you mention what you are doing?

Reply:
@julenka, per my PR comments, all TODO: Troys are still to be addressed. I just release the PR already to get things going. I'll update my PR summary to bold this point